spec/features/ncr/work_orders/create_spec.rb

```diff
@@ -159,6 +159,19 @@
       expect(page).to have_content('$2,000 for construction')
     end
 
+    scenario "selecting Expense Type toggles Building required flag", :js do
+      login_as(requester)
+      visit "/ncr/work_orders/new"
```

Comment:
`new_ncr_work_order_path` ? (I've been slowly moving specs over to that way of calling paths)